specs/v1/paths/api/service-city-zones.yaml

```diff
@@ -0,0 +1,119 @@
+get:
+  tags:
+    - Regions
+  summary: Retrieve service city zones
+  operationId: serviceCityGeographic areas
```

Comment:
Operation ID can't contain spaces can it?

Reply:
search replace error fixed

Reply:
> Operation ID can't contain spaces can it?

To answer the specific question though, it seems so, yes in the url it is changed to `%20`